python/paddle/v2/fluid/tests/book/test_label_semantic_roles.py

```diff
@@ -211,12 +211,74 @@ def main():
                 if batch_id != 0:
                     print("second per batch: " + str((time.time() - start_time)
                                                      / batch_id))
-
-            # exit early for CI
-            exit(0)
+                if save_dirname is None:
+                    exit(0)
+                # Set the threshold low to speed up the CI test
+                if float(pass_precision) > 0.1:
+                    fluid.io.save_inference_model(save_dirname, [
+                        'word_data', 'verb_data', 'ctx_n2_data', 'ctx_n1_data',
+                        'ctx_0_data', 'ctx_p1_data', 'ctx_p2_data', 'mark_data'
+                    ], [feature_out], exe)
+                    return
 
             batch_id = batch_id + 1
 
 
+def infer(save_dirname=None):
+    if save_dirname is None:
+        return
+
+    place = fluid.CPUPlace()
+    exe = fluid.Executor(place)
+
+    # Use fluid.io.load_inference_model to obtain the inference program desc,
+    # the feed_target_names (the names of variables that will be feeded 
+    # data using feed operators), and the fetch_targets (variables that 
+    # we want to obtain data from using fetch operators).
+    [inference_program, feed_target_names,
+     fetch_targets] = fluid.io.load_inference_model(save_dirname, exe)
+
+    data = [[0, 1, 0, 1], [0, 1, 1, 0, 0, 1]]
```

Comment:
I think we'd better use a more common data for inference. There are two optional methods.
1. Infer a real data like https://github.com/PaddlePaddle/book/blob/develop/07.label_semantic_roles/README.md#application , if the real testing data is small enough.
2. Infer a random data.

Reply:
I definitely agree.  I will check whether I will do it in this PR or in future PR.

Reply:
Added Infer a random integer data. Will add inferring real data in future PR.